build.gradle

```diff
@@ -29,9 +29,8 @@ ext {
 dependencies {
     api group: 'ch.poole', name: 'OpeningHoursParser', version: parserVersion
 
-    testImplementation  group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: junitVersion
-    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-params', version: junitVersion
-    testRuntimeOnly  group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junitVersion
+    testImplementation  group: 'org.junit.jupiter', name: 'junit-jupiter', version: junitVersion
+    testRuntimeOnly  group: 'org.junit.platform', name: 'junit-platform-launcher'
```

Comment:
Note how I also don't need to declare the junitVersion twice.

From the JUnit 5 user guide:

> Using the BOM allows you to omit the version when declaring dependencies on all artifacts with the org.junit.platform, org.junit.jupiter, and org.junit.vintage group IDs.
>
>Since all JUnit artifacts declare a [platform](https://docs.gradle.org/current/userguide/platforms.html) dependency on the BOM, you usually don’t need to declare an explicit dependency on it yourself. Instead, it’s sufficient to declare one regular dependency that includes a version number. Gradle will then pull in the BOM automatically so you can omit the version for all other JUnit 5 artifacts.